src/support.js

```diff
@@ -84,6 +84,29 @@ var asm2wasmImports = { // special asm2wasm imports
 };
 
 #if RELOCATABLE
+// dynamic linking
+var $LDSO = {
```

Comment:
What does LDSO stand for?

Reply:
It stands for `ld.so` which is known as "dynamic linker/loader" (see e.g. http://man7.org/linux/man-pages/man8/ld.so.8.html).